app/components/system_admin/schools/details/view.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module SystemAdmin
+  module Schools
+    module Details
+      class View < ViewComponent::Base
+        attr_reader :school_entity
+
+        def initialize(school_entity)
+          @school_entity = school_entity
+        end
+
+        def lead_partner?
+          if school_entity.is_a?(School)
+            school_entity.lead_school
```

Comment:
That is the `School#lead_school` column. I'm not sure if Adam has removed it but I think we should keep this for the time being since this data might be used for other purposes like for analytics.